Twig/DoctrineExtension.php

```diff
@@ -40,6 +40,7 @@ public function getFilters()
             new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery')),
             new \Twig_SimpleFilter('doctrine_pretty_query', 'SqlFormatter::format', array('is_safe' => array('html'))),
             new \Twig_SimpleFilter('doctrine_replace_query_parameters', array($this, 'replaceQueryParameters'), array('is_safe' => array('html'))),
```

Comment:
This method is actually not safe when highlighting is disabled.
I think it might be worth using a dedicated filter to highlight all the time